### ISSUE TITLE
cloning the repository did not work for me with command in readme

### ISSUE THREAD

**Issue by RaoulVanBergen** [SiliconLabs/wisun-br-linux-docker#7]
Hi 
in the readme it states following command to use for cloning the repository:
`git clone git@github.com:SiliconLabs/wisun-br-linux-docker.git`
this did not work for me (permission errors)
the following command worked fine:
`git clone https://github.com/SiliconLabs/wisun-br-linux-docker.git`
As I am not a git expert, I am not sure if I did something wrong, thought to report it anyway.
Thanks, Raoul

**Comment by jerome-pouiller** [comment]
Indeed, the command in README relies on ssh keys. URLs in `https://` don't have this issue. We are going to fix it.
